test/imports.ts

```diff
@@ -1,13 +1,11 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+/**
+ * @license Copyright 2016 Palantir Technologies, Inc. All rights reserved.
  */
 
-/*
-Quick explanation of what's going on here:
-
-This file imports every Blueprint package with JS exports to ensure that the typings can be consumed by various
-TypeScript project configurations. A Gulp task at the bottom of `gulp/typescript.js` attempts to compile this file
-in TypeScript 2.0 with `--strictNullChecks`.
+/**
+ * @fileoverview This file imports every Blueprint package with JS exports to ensure that the typings can be
```

Comment:
does this pragma get used anywhere? does any editor understand what to do with it?